src/atom.coffee

```diff
@@ -96,7 +96,7 @@ class Atom extends Model
   #
   # Returns the absolute path to ~/.atom
   @getConfigDirPath: ->
-    @configDirPath ?= fs.absolute('~/.atom')
+    @configDirPath ?= remote.process.env.ATOM_HOME
```

Comment:
Is there a reason you are using `remote.process.env.ATOM_HOME` instead of just `process.env.ATOM_HOME`?